src/react-share.jsx

```diff
@@ -2,10 +2,12 @@ import { generateIcon } from './generateIcon';
 import * as ShareButtons from './share-buttons';
 import * as ShareCounts from './share-counts';
 import * as shareLinks from './social-media-share-links';
+import * as shareCountGetters from './share-count-getters';
 
 export {
   generateIcon as generateShareIcon,
   ShareButtons,
   ShareCounts,
   shareLinks,
+  shareCountGetters,
```

Comment:
Why is this exported in this PR? I would not like to expose the shareCountGetters unless there's a very good reason :).